packages/node_modules/@ciscospark/recipe-private-web-client/src/config.js

```diff
@@ -2,15 +2,30 @@
  * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
  */
 
+/* eslint-env browser */
+
 import LocalForageStoreAdapter from '@ciscospark/storage-adapter-local-forage';
 import LocalStorageStoreAdapter from '@ciscospark/storage-adapter-local-storage';
+import SessionStorageStoreAdapter from '@ciscospark/storage-adapter-session-storage';
+
+let boundedAdapter;
+let unboundedAdapter;
+
+if (localStorage && localStorage.getItem('remember-me') === 'true') {
+  boundedAdapter = new LocalStorageStoreAdapter('web-client-internal');
+  unboundedAdapter = new LocalForageStoreAdapter('web-client-internal');
```

Comment:
Local storage will be ok if the user says to "remember me"?

Reply:
Yeah that's the gist. I wanted a way to keep the existing functionality since this change will be a less than stellar user experience. If the user "opts in" then it's their choice re: the security stance